projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -36,11 +39,36 @@ class TestComponent {
   open = false;
   disabled = false;
   showDescription = false;
+  showHeading = false;
+  headingLevel: HeadingLevel = 3;
```

Comment:
```suggestion
  headingLevel: HeadingLevel;
```

This change will fix the tests.